profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -205,6 +205,15 @@ function _tmgmt_poetry_form_load_translator_settings(&$form_state, &$form, $job)
     $settings = [];
   }
 
+  // NEPT-62: Overwriting the 'delai' field properties.
+  if (isset($settings['delai'])) {
+    $settings['delai']['#title'] = t('Requested delivery date');
+    $settings['delai']['#date_year_range'] = '-0:+1';
+    $settings['delai']['#default_value'] = '';
```

Comment:
also the default value is set previously and you overwrite it here to an empty string!